src/EnumerableSet.sol

```diff
@@ -0,0 +1,212 @@
+// SPDX-FileCopyrightText: © 2022 Dai Foundation <www.daifoundation.org>
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+// Adapted from: OpenZeppelin Contracts v4.4.1 (utils/structs/EnumerableSet.sol)
+pragma solidity ^0.8.14;
+
+/**
+ * @dev Library for managing
+ * https://en.wikipedia.org/wiki/Set_(abstract_data_type)[sets] of primitive
+ * types.
+ *
+ * Sets have the following properties:
+ *
+ * - Elements are added, removed, and checked for existence in constant time
+ * (O(1)).
+ * - Elements are enumerated in O(n). No guarantees are made on the ordering.
+ *
+ * ```
+ * contract Example {
+ *     // Add the library methods
+ *     using EnumerableSet for EnumerableSet.AddressSet;
+ *
+ *     // Declare a set state variable
+ *     EnumerableSet.AddressSet private mySet;
+ * }
+ * ```
+ *
+ * As of v3.3.0, sets of type `bytes32` (`Bytes32Set`), `address` (`AddressSet`)
+ * and `uint256` (`UintSet`) are supported.
+ */
+library EnumerableSet {
```

Comment:
Once again, this is not an enhanced `ilk-registry`, it's meant to store information that currently is not compatible with the `ilk-registry` and ends up polluting the Chainlog.

OpenZeppelin is constantly audited and, being largely used by many projects in the industry, is always under the scrutiny of the broader Ethereum community, so we feel comfortable adding `EnumerableSet` as a dependency.

Enumerability is a requirement for [our use cases](https://forum.makerdao.com/t/introducing-rwa-registry/19782), hence we opted for use OpenZeppelin's instead of rolling our own.